.circleci/bin/build-and-push-frontend.sh

```diff
@@ -5,6 +5,7 @@ set -e
 
 # env variables
 REACT_APP_BASE_URL=${REACT_APP_API_BASE_URL:-}
+REACT_APP_PUBLIC_KEY=${REACT_APP_PUBLIC_KEY:-}
```

Comment:
```suggestion
REACT_APP_FLUTTERWAVE_PUBLIC_KEY=${REACT_APP_PUBLIC_KEY:-}
```

Reply:
I suggest that we use more descriptive names for the variables. We may have more public keys in future.

Reply:
Thank you, I have made changes